### ISSUE TITLE
[Feature Proposal] Toggle read-only mode 

### ISSUE THREAD

**Issue by ellenchristine** [BoostIO/Boostnote#2298]
I like that we're able to easily toggle between the editor and editor+preview views. However, there are lots of notes I have that I only need to reference in read-only mode, so it would be helpful to have a read-only view that hides the Markdown editor entirely.

**Comment by modmod24** [comment]
i agree, but can you elaborate? how is the current behaviour interfering with your use case?

**Comment by matthaliski** [comment]
Hey @ellenchristine, I just started playing around with Boostnote and think you might find your answer by toggling this:

<img width="494" alt="screen shot 2018-08-17 at 12 17 19 pm" src="https://user-images.githubusercontent.com/431169/44284729-986c9300-a217-11e8-9140-ce5014af3f47.png">


It defaults to the read-only mode even after editing a different note. Perhaps this is what you're looking for. Additionally, there's a setting to require a double-click (or right mouse) to enter edit mode.

I had to play around a bit before I found it.

**Comment by ellenchristine** [comment]
Thanks @matthaliski! 

For anyone else having the issue, I had to change the "Switch to Preview" setting to "When Editor Blurred, Edit on Double Click." Then, while toggling to preview mode, I had to move the mouse away from the toggle _before_ mousing up. It's a little tricky at first, but now that I have the hang of it, I'm able to see my notes in preview-only mode.

**Comment by modmod24** [comment]
can this issue bee closed?
